client/sniffer/src/main/java/org/elasticsearch/client/sniff/ElasticsearchNodesSniffer.java

```diff
@@ -35,13 +35,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
These will not pass tests, we keep our imports explicit. You may have to change your IDE to not do this.

Reply:
ok, fixed.